File: app/src/main/java/sk/styk/martin/apkanalyzer/activity/detailfragment/AppDetailFragment_Service.java
package sk.styk.martin.apkanalyzer.activity.detailfragment;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import java.util.List;

import sk.styk.martin.apkanalyzer.R;
import sk.styk.martin.apkanalyzer.activity.AppDetailFragment;
import sk.styk.martin.apkanalyzer.adapter.ServiceListAdapter;
import sk.styk.martin.apkanalyzer.model.ServiceData;

/**
 * Created by Martin Styk on 30.06.2017.
 */
public class AppDetailFragment_Service extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.fragment_app_detail_service, container, false);

        List<ServiceData> data = getArguments().getParcelableArrayList(AppDetailFragment.ARG_CHILD);

        RecyclerView recyclerView = (RecyclerView) rootView.findViewById(R.id.recycler_view_service);

        RecyclerView.Adapter adapter = new ServiceListAdapter(data);
        recyclerView.setAdapter(adapter);
        recyclerView.setHasFixedSize(true);

        return rootView;
    }
}

File: app/src/main/java/sk/styk/martin/apkanalyzer/activity/detailfragment/AppDetailFragment_Service.kt
package sk.styk.martin.apkanalyzer.activity.detailfragment

import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v7.widget.RecyclerView
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import sk.styk.martin.apkanalyzer.R
import sk.styk.martin.apkanalyzer.activity.AppDetailFragment
import sk.styk.martin.apkanalyzer.adapter.detaillist.ServiceListAdapter
import sk.styk.martin.apkanalyzer.model.detail.ServiceData

/**
 * @author Martin Styk
 * @version 30.06.2017.
 */
class AppDetailFragment_Service : Fragment() {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val rootView = inflater!!.inflate(R.layout.fragment_app_detail_service, container, false)

        val data = arguments.getParcelableArrayList<ServiceData>(AppDetailFragment.ARG_CHILD)

        val recyclerView = rootView.findViewById<RecyclerView>(R.id.recycler_view_service)

        val adapter = ServiceListAdapter(data!!)
        recyclerView.adapter = adapter
        recyclerView.setHasFixedSize(true)

        return rootView
    }
}
